src/Command/ListenCommand.php

```diff
@@ -8,27 +8,36 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Yiisoft\Yii\Console\ExitCode;
 use Yiisoft\Yii\Queue\QueueFactory;
 use Yiisoft\Yii\Queue\QueueFactoryInterface;
 
-class ListenCommand extends Command
+final class ListenCommand extends Command
 {
     protected static $defaultName = 'queue/listen';
-    private QueueFactory $queueFactory;
+    private QueueFactoryInterface $queueFactory;
 
-    public function __construct(?string $name, QueueFactoryInterface $queueFactory)
+    public function __construct(QueueFactoryInterface $queueFactory)
     {
-        parent::__construct($name);
+        parent::__construct();
         $this->queueFactory = $queueFactory;
-        $this->setDescription('Listens the queue and executes messages as they come. Needs to be stopped manually.');
+    }
 
-        $this->addArgument('channel', InputArgument::OPTIONAL, 'Queue channel name to connect to', QueueFactory::DEFAULT_CHANNEL_NAME);
+    public function configure(): void
+    {
+        $this->setDescription('Listens the queue and executes messages as they come. Needs to be stopped manually.');
```

Comment:
To make command lazy move description to `protected static $defaultDescription`

Reply:
done

Reply:
`$this->setDescription(self::$defaultDescription)` is redundant